### PR TITLE
feat(cypress): Restructure and add Account Portal section

### PR DESCRIPTION
**Pull request by anagstef**
> [!IMPORTANT]
> 🔎 Previews:
>
> - https://clerk.com/docs/pr/1679/testing/cypress/test-account-portal
> - https://clerk.com/docs/pr/1679/testing/cypress/overview
> - https://clerk.com/docs/pr/1679/testing/cypress/custom-commands
> - https://clerk.com/docs/pr/1679/testing/overview

### This PR:

- Restructures the Cypress pages under the Testing docs
- Adds a new Test Account Portal mini guide

### CODE REVIEW

File: docs/testing/cypress/overview.mdx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  </CodeBlockTabs>
+
+  ### Set your API keys
+
+  In your test runner, set your publishable and secret key as the `CLERK_PUBLISHABLE_KEY` and `CLERK_SECRET_KEY` environment variables, respectively.
+
+  To find your keys:
+
+  1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=api-keys).
+  1. In the navigation sidebar, select **API Keys**.
```
Comment (alexisintech) [inline docs/testing/cypress/overview.mdx:42]:
```suggestion
  1. In the Clerk Dashboard, navigate to the [**API Keys**(https://dashboard.clerk.com/last-active?path=api-keys) page.
```

File: docs/testing/cypress/overview.mdx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    e2e: {
+      setupNodeEvents(on, config) {
+        return clerkSetup({ config })
+      },
+      baseUrl: 'http://localhost:3000', // your app's URL
+    },
+  })
+  ```
+
+  `clerkSetup` will obtain a [Testing Token](/docs/testing/overview#testing-tokens) when your test suite starts, making it available for all subsequent tests to use.
```
Comment (alexisintech) [inline docs/testing/cypress/overview.mdx:66]:
```suggestion
  `clerkSetup()` will obtain a [Testing Token](/docs/testing/overview#testing-tokens) when your test suite starts, making it available for all subsequent tests to use.
```

File: docs/testing/cypress/overview.mdx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+> Check out the [demo repo](https://github.com/clerk/clerk-cypress-nextjs) that tests a Clerk-powered application using [Testing Tokens](/docs/testing/overview#testing-tokens).
+
+<Steps>
+  ### Install `@clerk/testing`
+
+  Clerk's testing package provides integration helpers for popular testing frameworks. Install it by running the following command:
+
+  <CodeBlockTabs options={['npm', 'yarn', 'pnpm']}>
+    ```sh {{ filename: 'terminal' }}
+    npm i @clerk/testing --save-dev
```
Comment (alexisintech) [inline docs/testing/cypress/overview.mdx:23]:
```suggestion
    npm install @clerk/testing --save-dev
```

File: docs/testing/cypress/test-account-portal.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+- is redirected back to the protected page.
+
+```ts
+import { setupClerkTestingToken } from '@clerk/testing/cypress'
+
+describe('Testing Tokens', () => {
+  it('sign in with Account Portal redirects', () => {
+    setupClerkTestingToken()
+
+    cy.origin('https://ready-flea-75.accounts.dev', () => {
```
[the remark this answers is not in the document]
Comment (anagstef) [reply docs/testing/cypress/test-account-portal.mdx:55]: Can you advise how mock data should look like? 🙏

[the remark this answers is not in the document]
Comment (anagstef) [reply docs/testing/cypress/test-account-portal.mdx]: Thank you! Done! ✅

**Review by alexisintech — APPROVED** [review]

File: docs/manifest.json [hunk 5]
```diff
@@ -977,7 +977,26 @@
...
+                          "title": "Cypress",
+                          "collapse": true,
+                          "items": [
+                            [
+                              {
+                                "title": "Overview",
+                                "href": "/docs/testing/cypress/overview"
+                              },
+                              {
+                                "title": "Custom Commands",
```
Comment (alexisintech) [inline docs/manifest.json:990]:
```suggestion
                                "title": "Custom commands",
```

**Pull request merged**